dunamis-editor/src/components/DynaformEditor/PalletePanel.vue

```diff
@@ -1,49 +1,21 @@
 <template>
   <v-expansion-panels v-model="panel" accordion multiple>
-    <v-expansion-panel
-      class="caption"
-      v-for="(group, i) in paletteElements"
-      :key="i"
-    >
+    <v-expansion-panel class="caption" v-for="(group, i) in paletteElements" :key="i">
       <v-expansion-panel-header class="caption">{{
         group.label
       }}</v-expansion-panel-header>
       <v-expansion-panel-content elevation="0">
         <div class="vpm-drawer d-flex flex-column">
-          <draggable
-            v-model="group.elements"
-            :group="{
-              name: 'people',
-              pull: 'clone',
-              put: false,
-            }"
-            :sort="false"
-            class="flex-wrap"
-            elevation="0"
-            :clone="clone"
-          >
-            <div
-              v-for="(item, n) in group.elements"
-              :key="n"
-              class="pa-1 vpm-drawer-item-list"
-            >
-              <v-card
-                class="d-flex pa-1"
-                flat
-                :hover="true"
-                color="transparent"
-              >
-                <v-icon
-                  small
-                  color="accent"
-                  class="pe-3 vpm-pallete-icon"
-                  v-text="item.icon"
-                ></v-icon>
+          <draggable v-model="group.elements" :group="{
+            name: 'people',
+            pull: 'clone',
+            put: false,
+          }" :sort="false" class="flex-wrap" elevation="0" :clone="clone">
+            <div v-for="(item, n) in group.elements" :key="n" class="pa-1 vpm-drawer-item-list">
+              <v-card class="d-flex pa-1" flat :hover="true" color="transparent">
+                <v-icon small color="accent" class="pe-3 vpm-pallete-icon" v-text="item.icon"></v-icon>
```

Comment:
keep it in multiple lines, is most easy to read the code

Reply:
done